spec/pubid_itu/identifier_spec.rb

```diff
@@ -171,6 +171,12 @@ module Pubid::Itu
       it { expect(subject).to be_a(Identifier::RegulatoryPublication) }
     end
 
+    context "ITU-T G.780/Y.1351" do
+      let(:pubid) { "ITU-T G.780/Y.1351" }
```

Comment:
Can you also test the retrieval of the subseries and the second-number? Thanks.

Reply:
done